src/components/form/field-hint/field-hint.tsx

```diff
@@ -0,0 +1,23 @@
+import { FC } from 'react';
+import clsx from 'clsx';
+import { FieldHintProps } from './types';
+
+export const FieldHint: FC<FieldHintProps> = ({ id, hintText, errorText }) => {
+    if (!hintText && !errorText) return null;
```

Comment:
Maybe add enum for checking on variants (for example TextVariant.HINT and TextVariant.ERROR) and only one text prop.